src/app/AttributeValueEncoder.h

```diff
@@ -163,25 +168,26 @@ class AttributeValueEncoder
     friend class ListEncodeHelper;
     friend class TestOnlyAttributeValueEncoderAccessor;
 
+    // Returns true if the list item should be encoded.  If it should, the
+    // passed-in TLVWriter will be used to checkpoint the current state of our
+    // attribute report list builder.
+    bool ShouldEncodeListItem(TLV::TLVWriter & aCheckpoint);
+
+    // Does any cleanup work needed after attempting to encode a list item.
```

Comment:
```suggestion
    // Is any cleanup work needed after attempting to encode a list item.
```

Reply:
No, "Does" is correct.  This is not the predicate.  It's a function that does some work.